### PR TITLE
Added clone type in Mx9 documentation

### PR DESCRIPTION
**Pull request by tknippenberg**

### CODE REVIEW

**Comment by tknippenberg** [comment]
@blis-mendix , can you verify whether:
- Text about preferences dialog is correct
- "Convert to full clone..." is available in Mx9 or not

And perhaps a general quick scan if there are differences from the Mx10 docs (where I copied this from)

File: content/en/docs/refguide9/version-control/using-version-control-in-studio-pro/clone-type.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* Cloning goes faster: as only relevant data is downloaded you can get started very quickly.
+* Limited disk space usage: your local repository will be much smaller compared to a full clone.
+
+The disadvantage of this close type is that additional download may be needed: when [combining changes](/refguide/merge-algorithm/) from an older branch, more data will have to be downloaded compared to the full clone, which can take slightly longer. The same applies when interacting with the history.
+
+## Preferences
+
+You can change the [clone type](/refguide/preferences-dialog/#clone) for future clone operations, such as downloading an app or checking out another branch of an app you already downloaded. Changing this setting will not affect apps that you have already downloaded.
+
+You can also change this preference from the **Open App** dialog box selecting **Preferences** on the left.
```
Comment (blis-mendix) [inline content/en/docs/refguide9/version-control/using-version-control-in-studio-pro/clone-type.md]: On 9 you can't there is nothing like this.

**Review by blis-mendix — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide9/version-control/using-version-control-in-studio-pro/clone-type.md (hunk 1 above)
Comment (tknippenberg) [reply content/en/docs/refguide9/version-control/using-version-control-in-studio-pro/clone-type.md]: Feedback processed. Ready for review by Docs team.

**Comment by tknippenberg** [comment]
Ready for review by Docs team, ready to merge once reviewed.

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
@tknippenberg 
I've done a proofread of the document which you can review in these commits: https://github.com/mendix/docs/pull/8393/files/cfe004853c3b78123bda483d615b3bc62dcfae06..c15a5ede97fdffa04e487a22a4eb8e0a1b058c0e

Can you quickly check my changes. If they are good, I'll apply them to the Mx10 document as well and publish.
Thanks.

**Comment by tknippenberg** [comment]
> @tknippenberg I've done a proofread of the document which you can review in these commits: https://github.com/mendix/docs/pull/8393/files/cfe004853c3b78123bda483d615b3bc62dcfae06..c15a5ede97fdffa04e487a22a4eb8e0a1b058c0e
> 
> Can you quickly check my changes. If they are good, I'll apply them to the Mx10 document as well and publish. Thanks.

Went through the changes and it LGTM! Thank you!

**Review by MarkvanMents — APPROVED** [review]
Thanks for the PR.
Approving and merging.

**Pull request merged**